specification/applicationinsights/data-plane/LiveMetrics/preview/2024-04-01-preview/livemetrics.json

```diff
@@ -0,0 +1,1213 @@
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "instanceName"
+    },
+    "IsSubscribedOperationRequestParameters.invariantVersion": {
+      "name": "x-ms-qps-invariant-version",
+      "in": "header",
+      "description": "Version/generation of the data contract (MonitoringDataPoint) between the client and Live Metrics.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "invariantVersion"
+    },
+    "IsSubscribedOperationRequestParameters.machineName": {
+      "name": "x-ms-qps-machine-name",
+      "in": "header",
+      "description": "Computer name where Application Insights SDK lives. Live Metrics uses machine name with instance name as a backup.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "machineName"
+    },
+    "IsSubscribedOperationRequestParameters.monitoringDataPoint": {
+      "name": "monitoringDataPoint",
+      "in": "body",
+      "description": "Data contract between Application Insights client SDK and Live Metrics. /QuickPulseService.svc/ping uses this as a backup source of machine name, instance name and invariant version.",
+      "required": false,
+      "schema": {
+        "$ref": "#/definitions/MonitoringDataPoint"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "IsSubscribedOperationRequestParameters.roleName": {
+      "name": "x-ms-qps-role-name",
+      "in": "header",
+      "description": "Cloud role name of the service.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "roleName"
+    },
+    "IsSubscribedOperationRequestParameters.streamId": {
+      "name": "x-ms-qps-stream-id",
+      "in": "header",
+      "description": "Identifies an Application Insights SDK as trusted agent to report metrics and documents.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "streamId"
+    },
+    "IsSubscribedOperationRequestParameters.transmissionTime": {
+      "name": "x-ms-qps-transmission-time",
+      "in": "header",
+      "description": "Timestamp when the client transmits the metrics and documents to Live Metrics. A 8-byte long type of ticks.",
+      "required": false,
+      "type": "integer",
+      "format": "int64",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "transmissionTime"
+    },
+    "PublishOperationRequestParameters.apiKey": {
+      "name": "apiKey",
+      "in": "query",
+      "description": "An alternative for authentication, but it's now deprecated. Please utilize Microsoft Entra ID as the preferred option.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
As above, I think this parameter should just be omitted from this API definition.